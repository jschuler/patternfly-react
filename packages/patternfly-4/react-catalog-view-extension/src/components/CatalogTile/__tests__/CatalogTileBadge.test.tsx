// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { CatalogTileBadge } from '../CatalogTileBadge';

describe('CatalogTileBadge test', () => {
  it('CatalogTileBadge should match snapshot', () => {
    const view = shallow(
      <CatalogTileBadge
        children={<>ReactNode</>}
				className={"''"}
				id={"null"}
				title={"null"}
      />);
    expect(view).toMatchSnapshot();
  });
});
