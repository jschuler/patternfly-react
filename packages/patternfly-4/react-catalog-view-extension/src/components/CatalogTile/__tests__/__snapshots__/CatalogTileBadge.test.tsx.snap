// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CatalogTileBadge test CatalogTileBadge should match snapshot 1`] = `
<Fragment>
  <Tooltip
    appendTo={[Function]}
    aria="describedby"
    boundary="window"
    className=""
    content="null"
    distance={15}
    enableFlip={true}
    entryDelay={500}
    exitDelay={500}
    flipBehavior={
      Array [
        "top",
        "right",
        "bottom",
        "left",
        "top",
        "right",
        "bottom",
      ]
    }
    id="null"
    isAppLauncher={false}
    isContentLeftAligned={false}
    isVisible={false}
    maxWidth="18.75rem"
    position="top"
    tippyProps={Object {}}
    trigger="mouseenter focus"
    zIndex={9999}
  >
    <span
      className="catalog-tile-pf-badge ''"
    >
      ReactNode
      <span
        className="sr-only"
      >
        null
      </span>
    </span>
  </Tooltip>
</Fragment>
`;
